bin/cmd.js

```diff
@@ -76,6 +83,12 @@ if (argv.reporter) {
   })
 }
 
+var lintOpts = {}
+if (argv.es6)
```

Comment:
It's funny to me that the code style in this repo is different from the code that it's designed to validate, lol.